src/pages/vendor/VendorEditProductPage/CategoriesForm/index.tsx

```diff
@@ -0,0 +1,239 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box } from '@mui/system';
+
+import StyledButton from 'src/components/shared/StyledButton';
+import {
+  PaddingVariants,
+  StyleVariants,
+} from 'src/components/shared/StyledButton/types';
+import { CustomSelect } from 'src/components/shared/StyledSelect';
+import {
+  OnboardingHeader4,
+  OnboardingText,
+} from 'src/pages/OnboardingPage/styles';
+import {
+  setCategory,
+  setStyle,
+  setType,
+  increaseAddProductStep,
+  decreaseAddProductStep,
+} from 'src/redux/addProduct/addProductSlice';
+import { useAppDispatch, useAppSelector } from 'src/redux/hooks';
+import theme from 'src/theme';
+
+import RejectEditingModal from '../RejectEditingModal';
+
+const productCategories = [
+  { label: 'Accessories', value: 'accessories' },
+  { label: 'Bags', value: 'bags' },
+  { label: 'Clothing', value: 'clothing' },
+  { label: 'Shoes', value: 'shoes' },
+  { label: 'Designers', value: 'designers' },
+  { label: 'Evental', value: 'evental' },
+];
+
+const clothesTypes = [
+  { label: 'Shoes', value: 'shoes' },
+  { label: 'Dress', value: 'dress' },
+  { label: 'Bag', value: 'bag' },
+  { label: 'Jeans', value: 'jeans' },
+  { label: 'Accessory', value: 'accessory' },
+  { label: 'Other', value: 'other' },
+];
+
+const styles = [
+  { label: 'Casual', value: 'casual' },
+  { label: 'Premium', value: 'premium' },
+  { label: 'Fancy', value: 'fancy' },
+];
+
+function CategoriesForm() {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const selectedCategories = useAppSelector(
+    (state) => state.addProduct.categories
+  );
+  const selectedType = useAppSelector((state) => state.addProduct.type);
+  const selectedStyle = useAppSelector((state) => state.addProduct.style);
+
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+
+  const toggleModal = (): void => {
+    setIsModalOpen(!isModalOpen);
+  };
+
+  const [clothesCategory, setClothesCategory] = useState(selectedCategories[0]);
```

Comment:
types

Reply:
fixed in https://github.com/ZenBit-Tech/code-lions_fe/pull/63/commits/d2b2d00bc14bc6a955a3c018e9d01fd2958c153e